contexts/credentials/v2

```diff
@@ -64,7 +64,58 @@
     "...": {
       "@id": "https://www.iana.org/assignments/jwt#..."
     },
-
+    "keys": {
+      "@id": "https://www.iana.org/assignments/jose#keys",
+      "@type": "@json"
+    },
+    "JsonWebKey": {
+      "@id": "https://w3id.org/security#JsonWebKey",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "controller": {
+          "@id": "https://w3id.org/security#controller",
+          "@type": "@id"
+        },
+        "publicKeyJwk": {
+          "@id": "https://w3id.org/security#publicKeyJwk",
+          "@type": "@json"
+        },
+        "secretKeyJwk": {
+          "@id": "https://w3id.org/security#secretKeyJwk",
+          "@type": "@json"
+        }
```

Comment:
A VC shouldn't be expressing private key material.

Reply:
^ this is not in the scope of the the "VerifiableCredential" RDF class.

... a VC can express RDF, if we want a VC's concept of "Multikey" or "JsonWebKey" to be the same, they need to have the same defintion.

Reply:
The definitions need to be consistent between what's in the broken out contexts and this one to ensure their compatibility.